src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -41,7 +48,8 @@ public static IServiceCollection AddApplicationInsights(this IServiceCollection
             return services;
         }
 
-        public static IServiceCollection AddApplicationInsights(this IServiceCollection services)
+        internal static IServiceCollection AddApplicationInsights(this IServiceCollection services, Action<ApplicationInsightsLoggerOptions> configureOptions,
+            Action<TelemetryConfiguration> additionalConfiguration)
```

Comment:
My understanding is that they're actions so that the webjobs sdk doesn't need to know about the specific classes implementing TelemetryConfiguration and ApplicationInsightsLoggerOptions in the host.